treeherder/model/models.py

```diff
@@ -616,6 +616,8 @@ class FailureLine(models.Model):
     id = BigAutoField(primary_key=True)
     job_guid = models.CharField(max_length=50)
     repository = models.ForeignKey(Repository)
+    job_log_id = models.BigIntegerField(null=True)  # This will become a foreign key once job_log is moved to this database
```

Comment:
Well, I just implemented enough of that migration to verify that from Django's point of view it isn't a problem. It might, however, represent excessive database churn so I'm happy to do it in the other order if we can make that happen in the next day or two.